dedibox-console/classic-hosting/how-to/configure-php.mdx

```diff
@@ -274,70 +266,62 @@ Scaleway offers different PHP versions:
 * The version 7.2
 * The version 7.3 
 
-
 <Message type="important">
   Versions prior to PHP 5.6 are deprecated, so we recommend you update the code of your website. If you need an older version, you must contact the support and request it.
 </Message>
 
 The default version for `.php` is 7.3 for current webhosting accounts. For older accounts the version can be changed directly from the [Dedibox console](https://console.online.net):
 
-1. Log into your account. 
-2. Click **Hosting**, then **Manage** next your webhosting. 
-3. Click **Manage** in the menu on the right. The **Website configuration** page displays. 
-4. Click the **Edit** action. 
-    <Lightbox src="scaleway_website_php.webp" alt="" />
-5. Select the PHP version of your choice.  
-    <Lightbox src="scaleway_php_versions.webp" alt="" />
-6. Click **Submit** to update your PHP version. 
+1. Log into your account.
+2. Click **Hosting**, then **Manage**, next to your webhosting.
+3. Click **Manage** in the menu on the right. The **Website configuration** page displays.
+4. Click the **Edit** action.
+    <Lightbox src="scaleway_website_php.webp" />
+5. Select the PHP version of your choice.
+6. Click **Submit** to update your PHP version.
 
 ## How to configure PHP 
 
-
 <Message type="important">
   Configuring PHP is possible for all our offers. However, ensure you are running at least PHP version 5.3 and above.
 </Message>
 
-
-1. Upload a `.user.ini`file to the folder corresponding to your sub domain (folder `/www` for example).
+1. Upload a `.user.ini`file to the folder corresponding to your subdomain (folder `/www` for example).
     <Message type="note">
       The majority of options are configurable, except for those with an impact on the resources of the server.
     </Message>
-2. You can modify all values with `PHP_INI_ALL` changeable option you can find by clicking this [link](https://www.php.net/manual/en/ini.list.php). 
+2. You can modify all values with `PHP_INI_ALL` changeable option you can find by clicking this [link](https://www.php.net/manual/en/ini.list.php).
     <Message type="important">
       - chown: modifications of the owner of a file are prohibited. Files uploaded using PHP or FTP already belong to your user.
       - system/exec/popen: the execution of binaries or CGI files is not possible for security reasons.
       - ASP is not supported. The execution of CGI scripts (cgi-bin) and other programs is disabled.
     </Message>
 
-
 ## How to solve a 500 Error in the htaccess file
```

Comment:
```suggestion
## How to solve a HTTP Error 500 in the htaccess file
```
not sure, wdyt?

Reply:
```suggestion
## How to solve an HTTP error 500 in the htaccess file
```

Reply:
I've added a second suggestion, just to fix a/an HTTP